protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json

```diff
@@ -6,18 +6,14 @@
       "adjustByMarket": "USDT-USD"
     },
     {
-      "exchangeName": "Bitstamp",
-      "ticker": "BCH/USD"
+      "exchangeName": "Bybit",
+      "ticker": "BCHUSDT",
+      "adjustByMarket": "USDT-USD"
     },
     {
       "exchangeName": "CoinbasePro",
       "ticker": "BCH-USD"
```

Comment:
I see the following, which suggests that we were supposed to update. Was this already our setting before research's suggestion?


![Screenshot 2023-10-04 at 3 45 07 PM](https://github.com/dydxprotocol/v4-chain/assets/19664986/9a9abed2-aa49-4c81-9ec8-7dbd298dfb51)

```
Coinbase_Pro |   | BCH | USDT | Remove
Coinbase_Pro |   | BCH | USD | Add


```

Reply:
Similarly for `Kraken`

Reply:
Yes, they gave us some updates that were already applied.

Reply:
Thank you for the close reading!

Reply:
If you check the "price diff oracles progress" page, you'll see where I marked some updates were already applied.